src/modules/api.js

```diff
@@ -1,12 +1,11 @@
-const API = async() => {
-try {
-  const apiUrl = ('https://restcountries.com/v3.1/all');
-  const response = await(apiUrl);
-  const data = await response.json();
-  console.log(data);
-}
-catch (error) {
-  console.log('Error',error);
- }
-}
-export default API;
\ No newline at end of file
+const API = async () => {
+  try {
+    const apiUrl = 'https://restcountries.com/v3.1/all';
+    const response = await apiUrl;
+    const data = await response.json();
+    console.log(data);
```

Comment:
- Hey, you still have the `console.log()` in this line. since you are just calling the value to ensure you await the result, you can bypass using console.log like this. Update the code to 

```
  const response = await apiUrl;
  await response.json();
```

Your code should still work fine :+1: